src/main/java/sorald/Main.java

```diff
@@ -71,6 +76,20 @@ public JSAP defineArgs() throws JSAPException{
 		opt.setHelp("Max number of fixes per rule. Default: Integer.MAX_VALUE (or all)");
 		jsap.registerParameter(opt);
 
+		opt = new FlaggedOption(Constants.ARG_MAX_FILES_PER_SEGMENT);
+		opt.setLongFlag(Constants.ARG_MAX_FILES_PER_SEGMENT);
+		opt.setStringParser(JSAP.INTEGER_PARSER);
+		opt.setDefault("" + 6500);
+		opt.setHelp("Max number of files per loaded segment for segmented repair. It should be >= 3000 files per segment. Default: 6500 (256mb Jvm) . ");
```

Comment:
Henry, you don't need to write the default value in the message, the default value is already shown automatically (this also needs to be fixed in the other option below). I already told you that in the original PR. Please, you need to start to pay attention on the stuff I say because I will not continue repeating them.